src/frontend/components/LiveVideoPanel/LiveVideoTabPanel/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Tab, Text } from 'grommet';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { LivePanelItem } from 'data/stores/useLivePanelState';
+import { ShouldNotHappen } from 'utils/errors/exception';
+import { theme } from 'utils/theme/theme';
+
+interface StyledTabProps {
+  selected?: boolean;
+}
+
+const colors = {
+  active: theme.global.colors['blue-active'],
+  focus: theme.global.colors['blue-focus'],
+  disable: theme.global.colors['blue-off'],
+  counterPart: theme.global.colors.white,
+};
+
+const StyledTab = styled(Tab)`
+  background: ${colors.counterPart};
+  padding: 3px 10px;
```

Comment:
please order property in alphabetical order

Reply:
do you agree to still place selector at the end ?

Reply:
https://github.com/openfun/marsha/pull/1248/commits/a9b856d179526fac42707213d5fcd03b6d404348